### ISSUE TITLE
Issue on Officer page

### ISSUE THREAD

**Issue by SurbhiBakshi** [cityofaustin/atd-data-tech#1329]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/aa04199a-2883-4e4f-9d3c-92ce48e10220)

**Comment by SurbhiBakshi** [comment]
Don't see this issue with the last test instance, am checking to see what the difference is. 
Also wrote to Knack about this.

**Comment by SurbhiBakshi** [comment]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/356ffd0f-ace8-4610-bc83-c35c74e0978e)

**Comment by SurbhiBakshi** [comment]
Hey Surbhi,
Thanks for your patience while we took a closer look here. From taking a closer look at the data source for your Calendar view - I'm seeing that something must have happened at some point to cause this view to break. The data source is currently showing as: Displaying officer_assignment records. connected to this page's PARENT record. here: https://builder.knack.com/atd/vza#pages/scene_82/views/view_142

This usually only occurs when a connection between two objects that a view relied on was changed or deleted. To fix this, you'll need to re-build the view to work with the new connections.
Let me know if you have any questions.

--

 
Valerie from Knack
